infrastructure/parameters/prod.parameters.json

```diff
@@ -46,6 +46,15 @@
     },
     "storageAccountPrefix": {
       "value": "sa"
+    },
+    "minCapacityStatisticsDb": {
+      "value": "4"
+    },
+    "maxCapacityStatisticsDb": {
+      "value": 10
```

Comment:
Is this correct because the current space in use is 62Gb and the max storage size is currently 250Gb.